src/js/engagement_view/.yarnrc.yml

```diff
@@ -10,4 +8,4 @@ installStatePath: .yarn/state/install-state.gz
 
 nodeLinker: node-modules
 
-yarnPath: .yarn/releases/yarn-berry.cjs
+yarnPath: .yarn/releases/yarn-3.1.1.cjs
```

Comment:
I think you want to just remove this line as well since it's no longer being sourced from that location.

Reply:
So while true that the docker container has the yarn version set correctly, anyone running yarn locally won't necessarily be using the right yarn version. So far, this seems to be the preferred way to set the version although I do agree its stupid to basically have this twice. 
If you know of a config option I've missed to force local development to use the right yarn version please let me know since its been a while since I've done serious yarn spelunking